roadmap.md

```diff
@@ -19,28 +18,30 @@ Technology Roadmaps
 
 Architecture:
 
-- [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md)
```

Comment:
This section was created for new components. It's not intended as a long-term listing of architectural docs (otherwise, it's quite incomplete). These components are either now part of our docs or scenario-specific (CoreRT). I think we can remove the section.